integration-testing/config-local-host.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+# install njs
+sudo apt install -y curl gnupg2 ca-certificates lsb-release ubuntu-keyring
+
+curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor \
```

Comment:
after this PR see if you can get it running on OSX and Windows.
timebox it

Reply:
https://github.com/nginx/njs/issues/320#issuecomment-645199347.
This doesn't support windows :(

Reply:
Maybe we can have a ec2 host a server and can be accessed within a VPC, so that, we can have another windows ec2 run the client to test against it.

It's not local host, but more reliable than public internet.

Reply:
well, at least add a Mac run :|